plugins/kubernetes/app/models/kubernetes/client_config_file.rb

```diff
@@ -21,6 +21,10 @@ def client_for(context_name)
       contexts[context_name].try(:client)
     end
 
+    def extension_client_for_client_for(context_name)
```

Comment:
Kudos on the method name :) Since it's not used anywhere I presume this is some rubbish code?